test/test_images/transformevents/main.go

```diff
@@ -31,11 +31,15 @@ type example struct {
 }
 
 var (
-	msgPostfix string
+	eventType   string
+	eventSource string
+	eventData   string
 )
 
 func init() {
-	flag.StringVar(&msgPostfix, "msg-postfix", "", "The postfix we want to add for the message.")
+	flag.StringVar(&eventType, "event-type", "knative.eventing.test.e2e", "The Event Type to use.")
+	flag.StringVar(&eventSource, "event-source", "", "Source URI to use. Defaults to the current machine's hostname")
```

Comment:
Does this default?

Reply:
Sry I missed it when I copied from elsewhere. Added the default.